.github/workflows/ci.yml

```diff
@@ -265,17 +347,15 @@ jobs:
         uses: actions/checkout@v3
       - name: Merge built artifacts
         uses: actions/download-artifact@v3
-        with:
-          path: artifacts
       - name: Upload release
         env:
           TAG_NAME: ${{ github.ref_name }}
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          mkdir release
-          cp artifacts/jq-linux-ubuntu-22.04-gcc/jq release/jq-linux-amd64
-          cp artifacts/jq-macos-macos-13-gcc/jq release/jq-macos-amd64
-          cp artifacts/jq-windows-windows-2022-gcc/jq.exe release/jq-windows-amd64.exe
-          cp artifacts/jq-dist/jq-* release/
+          mkdir -p "release"
+          for files in jq-*; do
+              cp -rf "${files}/*" "release/"
```

Comment:
Bash does not expand wildcard in `"${files}/*"`.

Reply:
Nice find. Addressed in https://github.com/jqlang/jq/pull/2665/commits/8b486ab861db64efd5f9b65e8e26a815cb72feec